Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/Surveys.kt

```diff
@@ -49,7 +49,9 @@ class Surveys @Inject constructor(
     }
 
     suspend fun resetSurvey(type: Type) {
-        // TODO
+        if (type == Type.HIGH_RISK_ENCOUNTER) {
+            settings.clear()
```

Comment:
Shouldn't the flow be: OTPRepos clear method calls SurveySettings clear method?

Reply:
Yes! Thank you. Changed.